tests/scripts/run-xunit-perf.cmd

```diff
@@ -27,6 +27,8 @@ setlocal ENABLEDELAYEDEXPANSION
   set BETTER=desc
   set OPT_LEVEL=full_opt
   set VALID_OPTLEVELS=min_opt full_opt tiered
+  set SLICE_DIR=
```

Comment:
Can you make sure that we update the usage with the meaning of this command line argument

Reply:
Yep